packages/x6-plugin-selection/src/index.ts

```diff
@@ -47,6 +47,12 @@ export class Selection
       Selection.defaultOptions,
       options,
     )
+
+    if (!Array.isArray(this.options.eventTypes)) {
+      // a new option, compatible with the behavior of older versions
+      this.options.eventTypes = ['leftMouseDown', 'mouseWheelDown']
+    }
```

Comment:
due to unexpected `merge` method, I have to use this to keep the behavior works the same as before